### ISSUE TITLE
TOPIC: Empty gap when info isn't specified

### ISSUE THREAD

**Issue by BeccaMelhuish** [citizenos/citizenos-fe#485]
@kevincrepin @ilmartyrk what do we do about this? Not great to have them empty I think? 

Can we hide them when not filled in, or is that complicated @ilmartyrk?  

![image](https://github.com/citizenos/citizenos-fe/assets/61320526/8be7f629-8dff-44ad-bdf2-5030c838ea17)

**Comment by kevincrepin** [comment]
I agree with hiding or having N/A or "not defined" or smt

**Comment by BeccaMelhuish** [comment]
Or the ideal (phase II thing I guess) - hide them to participants when not filled in, but for admins/editors have an 'Add location' / 'Add language' link that opens settings in the right place?

**Comment by BeccaMelhuish** [comment]
But yes quick fix could be:  
Location not specified
Language not specified
Or too wordy?

**Comment by kevincrepin** [comment]
I think it's ok in English. Would need to see in other languages.

**Comment by BeccaMelhuish** [comment]
@oksks what do you think about this? :)

**Comment by BeccaMelhuish** [comment]
Though @ilmartyrk, the ideal (if I'm not mistaken @kevincrepin?) would be to hide them if they're empty. Would this be difficult to do?

**Comment by oksks** [comment]
_(From web experience, it's not a very complex task to have these options hidden unless filled, but app dev might be different)_
IMHO, definitely best to hide rather than add any texts I think, which would just be clutter from user's POV. And would look worse in other languages - just ran through a few languages I know in my head.

Also just started writing that I feel this whole box looks off for some reason, then went to check Figma just in case, and the design was overall with a different look initially:

Figma:
<img width="373" alt="Screenshot 2023-12-04 at 23 33 45" src="https://github.com/citizenos/citizenos-fe/assets/34126353/b5110638-bd6b-468f-b977-133c0acfa1d4">

vs

Actual:
<img width="310" alt="Screenshot 2023-12-04 at 23 34 24" src="https://github.com/citizenos/citizenos-fe/assets/34126353/e735b03f-345b-493a-bda7-a91b56dc58cc">

**Comment by kevincrepin** [comment]
@oksks alrdy mentioned this also in https://github.com/citizenos/citizenos-fe/issues/510 ;)

I'll give a counterargument to hiding it. Keeping it visible means people get used to the possibility of these variables. If we hide them, it hinders this learning. So I would still keep them. It will keep this box consistent in size as well. We can just have a label "Not specified"?

**Comment by anettlinno** [comment]
Agree that design-wise it´s better to keep all fields there. Let´s leave only "Not specified", then it shouldn´t go too long, @oksks?

**Comment by ilmartyrk** [comment]
@kevincrepin @oksks @anettlinno @BeccaMelhuish added text "Not specified", though I am not sure what to do with groups or categories when there are none. Currently if you click on them the arrow direction changes but there is no content.

**Comment by anettlinno** [comment]
@ilmartyrk I think current solution is decent. Let´s leave it like that until we have better idea :) 
Closing this issue.
